docs/options.md

```diff
@@ -85,3 +85,20 @@ To change the entry point you can use the option `-entry-point=FUNCTION_NAME`, w
 ## Calls to `klee-assume`
 
 By default, KLEE will report an error if the assumed condition is infeasible. The option `-silent-klee-assume` can be used to sliently terminate the current path exploration in such cases.
+
+## Statistics
+
+By default, KLEE generates two files containing statistics concerning the code exploration:
+
+* **run.stats**: This is a text file containing various statistics emitted by KLEE. While this file can be inspected manually, you should use the [klee-stats]({{site.baseurl}}/docs/tools) tool for that.
+* **run.istats**: This is a binary file containing global statistics emitted by KLEE for each line of code in the program. The file can be read with the callgring tool.
```

Comment:
"binary file" -> "text file" using "callgrind format".
Those files can be read with frontends, which are able to read it (e.g. https://kcachegrind.github.io/)